samples/qemuarm64.yml

```diff
@@ -6,7 +6,8 @@ machine: qemuarm64-thistle
 distro: thistle-base
 
 thistle-features:
-  meta-thistle: e05a0ab0e3abfc3c8fcb5371fdffa451765826af
+  meta-thistle: b90acb0a89eab8e123416e437f1f5f2186788f07
```

Comment:
Will update this revision value to a commit in the `main` branch of `meta-thistle`, once https://github.com/thistletech/meta-thistle/pull/14 is merged.